driver-core/src/main/java/com/datastax/driver/core/policies/LatencyAwarePolicy.java

```diff
@@ -367,20 +371,45 @@ public long getMeasurementsCount() {
         }
     }
 
+    /**
+     * A set of DriverException subclasses that we should prevent from updating the host's score.
+     * The intent behind it is to filter out "fast" errors: when a host replies with such errors,
+     * it usually does so very quickly, because it did not involve any actual
+     * coordination work. Such errors are not good indicators of the host's responsiveness,
+     * and tend to make the host's score look better than it actually is.
+     */
+    private static final Set<Class<? extends DriverException>> EXCLUDED_EXCEPTIONS = ImmutableSet.of(
```

Comment:
I hesitated a lot about `QueryValidationException`: my reasoning for not ignoring them was that an invalid query would most likely be invalid on all nodes, and we can expect that the failing queries would be evenly distributed, therefore not favoring a particular host's score.
But the fact that they don't change the relative scores can be an argument to ignore them too.
Anyway a stable deployment should have very few of these, so I'm fine with ignoring them.